<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Öğrenci Bilgilerini Alma Arayüzü</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="grp_egitim">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="title">
    <string>Eğitim</string>
   </property>
   <widget class="QRadioButton" name="rd_lisans">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Lisans</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_lisansustu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Lisansüstü</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_sinif">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Sınıf</string>
   </property>
   <widget class="QRadioButton" name="rd_sinif1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>1.Sınıf</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_sinif2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>2.Sınıf</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_sinif3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>3.Sınıf</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_sinif4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>4.Sınıf</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_zorunluderslerlisans">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Alınan Zorunlu Dersler</string>
   </property>
   <widget class="QCheckBox" name="cb_aiit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>AİİT</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_turkdili">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Türk Dili</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_isg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>İSG</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_durum">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Durum</string>
   </property>
   <widget class="QRadioButton" name="rd_ders">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ders</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_tez">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tez</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_zorunluderslerlisansustu">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>120</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Alınan Zorunlu Dersler</string>
   </property>
   <widget class="QCheckBox" name="cb_secmeli">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Seçmeli Dersler</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_seminerler">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Seminer</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_tez">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tez</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>82</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string>Öğrenci Ad : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string>Öğrenci Soyad : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string>Öğrenci Bölüm : </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>135</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="ln_ad"/>
    </item>
    <item>
     <widget class="QLineEdit" name="ln_soyad"/>
    </item>
    <item>
     <widget class="QLineEdit" name="ln_bolum"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>112</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="btn_bilgileriyazdir">
      <property name="text">
       <string>Bilgileri Yazdır</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_mezuniyetsorgula">
      <property name="text">
       <string>Mezuniyet Sorgula</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_ogrencidurumyazdir">
      <property name="text">
       <string>Öğrenci Durum Yazdır</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
